AppCenter/AppCenter/Internals/Ingestion/MSIngestionCall.m

```diff
@@ -68,28 +73,34 @@ - (void)resetRetry {
   [self resetTimer];
 }
 
-- (void)      ingestion:(id <MSIngestionProtocol>)ingestion
-callCompletedWithStatus:(NSUInteger)statusCode
-                   data:(nullable NSData *)data
-                  error:(NSError *)error {
+- (void)ingestion:(id<MSIngestionProtocol>)ingestion
+    callCompletedWithStatus:(NSUInteger)statusCode
+                       data:(nullable NSData *)data
+                      error:(NSError *)error {
   BOOL internetIsDown = [MSIngestionUtil isNoInternetConnectionError:error];
-  BOOL couldNotEstablishSecureConnection = [MSIngestionUtil isSSLConnectionError:error];
+  BOOL couldNotEstablishSecureConnection =
+      [MSIngestionUtil isSSLConnectionError:error];
 
   if (internetIsDown || couldNotEstablishSecureConnection) {
 
-    // Reset the retry count, will retry once the (secure) connection is established again.
+    // Reset the retry count, will retry once the (secure) connection is
+    // established again.
```

Comment:
Should we use `/* */` in the places like this?

Reply:
Ideally, yes. Will this work with default LLVM format?